src/documents/js/main.js

```diff
@@ -51,7 +51,8 @@ $(function() {
       return;
     }
     var codeEl = getCodeElement(this);
-    var code = codeEl.text();
+    // need to wrap in async IIFE for examples using `await` in top-level
+    var code = `(async() => { ${codeEl.text()} })()`; 
```

Comment:
this fixes running examples like these: https://jsforce.github.io/document/#query

![Screenshot 2024-07-11 at 7 54 18 PM](https://github.com/user-attachments/assets/05ae0ff1-874c-4795-b863-a99218485948)